flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "booker-flake";
+
+  inputs = { 
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { 
+          inherit system; 
+        };
+
+        buildNodeJs = pkgs.callPackage "${nixpkgs}/pkgs/development/web/nodejs/nodejs.nix" {
+          python = pkgs.python3;
```

Comment:
why do we need python to build node?

Reply:
because the creators of nodejs decided that python should be a dependency